app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
I've probably missed the discussions about this, but I'm curious why aren't we reusing the way we're doing filters for G-Cloud. We have a way to define forms using manifests and the logic to group options based on question type and it feels like it would be a more extensible approach, especially if we start filtering by brief fields in the future.